src/commands/appdistribution-distribute.ts

```diff
@@ -43,13 +50,42 @@ export const command = new Command("appdistribution:distribute <release-binary-f
     "--groups-file <file>",
     "path to file with a comma separated list of group aliases to distribute to",
   )
+  .option(
+    "--test-devices <string>",
+    "semicolon-separated list of devices to run automated tests on, in the format 'model=<model-id>,version=<os-version-id>,locale=<locale>,orientation=<orientation>'. Run 'gcloud firebase test android|ios models list' to see available devices. Note: This feature is in beta.",
```

Comment:
I don't love that this feature tells users to go use gcloud for this. I assume that listing available models is basically a single API call to an API that we've already implemented a client for. Why not just add an equivalent command to firebase-tools?

Reply:
Addressed this in [go/atad-cli-api-review](http://go/atad-cli-api-review).